### PR TITLE
pass the `standalone` request arg in the /caravel/slices/<slice_id>/ endpoint redirect

### PR DESCRIPTION
**Pull request by williaster**
Gave some context in slack, but this will be useful for us to auto-generate standalone viz's from a slice's `id`. This endpoint currently re-directs to a slice's url, but ignores all arguments passed to it.

@mistercrunch

### CODE REVIEW

**Comment by williaster** [comment]
how can I fix the coveralls blocker? :(

File: tests/core_tests.py [hunk 1]
```diff
@@ -147,9 +147,14 @@ def test_slices(self):
         for slc in db.session.query(Slc).all():
             urls += [
                 (slc.slice_name, 'slice_url', slc.slice_url),
-                (slc.slice_name, 'slice_id_endpoint', '/caravel/slices/{}'.format(slc.id)),
                 (slc.slice_name, 'json_endpoint', slc.viz.json_endpoint),
                 (slc.slice_name, 'csv_endpoint', slc.viz.csv_endpoint),
+                (slc.slice_name, 'slice_id',
```
Comment (mistercrunch) [inline tests/core_tests.py]: no need to test these 3 endpoints for all slices, let's pick one and write another unit test that tests using `assertRedirects`

File: caravel/views.py [hunk 2]
```diff
@@ -1141,7 +1141,9 @@ def slice(self, slice_id):
         qry = session.query(models.Slice).filter_by(id=int(slice_id))
         slc = qry.first()
         if slc:
-            return redirect(slc.slice_url)
+            url = '{slc.slice_url}&standalone={standalone}'.format(
+                slc=slc, standalone=request.args.get('standalone', False))
```
Comment (mistercrunch) [inline caravel/views.py]: `False` -> `'false'`

**Comment by williaster** [comment]
@mistercrunch I tried messing with the `assertRedirects` but wasted a ton of time trying to get the `flask_testing.TestCase` to work. This is the current best standard for how you're testing redirects so I went with it for now.

Also updated the `'false'` vs `False` logic.

**Comment by mistercrunch** [comment]
Nice!

**Pull request merged**